spring-ai-core/src/main/java/org/springframework/ai/chat/client/advisor/VectorStoreChatMemoryAdvisor.java

```diff
@@ -209,10 +219,25 @@ public Builder withSystemTextAdvise(String systemTextAdvise) {
 			return this;
 		}
 
+		public Builder withConversationId(String conversationId) {
```

Comment:
@ogbozoyan Thanks for your PR. These methods are already part of the AbstractChatMemoryAdvisor.AbstractBuilder<VectorStore> and hence they don't need to be added here. I will remove these but merge your PR with the deprecated constructors.